transportable-udfs-trino-plugin/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'java'
+apply plugin: 'distribution'
+
+java {
+  toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+}
+
+dependencies {
+  implementation project(':transportable-udfs-api')
+  implementation project(":transportable-udfs-trino")
+  implementation(group:'io.trino', name: 'trino-main', version: project.ext.'trino-version') {
+    exclude 'group': 'com.google.collections', 'module': 'google-collections'
+  }
+  implementation(group:'io.trino', name: 'trino-main', version: project.ext.'trino-version', classifier: 'tests') {
```

Comment:
we shouldn't depend on the 'tests' package in our implementation right? this can be a `testImplementation` dependency

Reply:
Done.